app/views/admin/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Admin Dashboard</h1>
+
+<%= link_to 'Merchants', '/admin/merchants' %>
+<%= link_to 'Invoices', '/admin/invoices' %>
+
+<h3> Top Customers </h3>
+<ol>
+<% @top_customers.each do |customer| %>
+  <li><%= customer[0] %> - <%= customer[1] %> purchase(s)</li>
+<% end %>
+</ol>
+
+<h3> Incomplete Invoices </h3>
+<% @pending_invoices.each do |invoice| %>
+  <div id="id-<%= invoice.id %>">
+    Invoice #<%= link_to "#{invoice.id}", "/admin/invoices/#{invoice.id}" %> - <%= invoice.created_at.strftime("%A, %B %-d, %Y") %><br>
+  </div>
+<% end %>
```

Comment:
it might be nice to wrap the top customers and incomplete invoices sections into `<div>` blocks so that we can test for them within those specific areas of the page. nbd though! that's just what we did for the merchants dashboard